documentation/user_docs/docs/manual/Special_sqw_information.rst

```diff
@@ -48,6 +48,64 @@ an ``sqw`` or a ``dnd`` object. It also returns general information about the co
 this object, i.e. data ranges, number of pixels in ``sqw`` file etc., whatever developers decided 
 most important for user to know about correspondent object.
 
+Note, that output from calling ``head(filename);`` and ``res = head(filename)`` differs:
+
+>> ``head(filename);`` **"Scientific" view:**
```

Comment:
I don't understand why this one is called scientific and the next one is called programmers. They are different uses of the function; one gives printed output to the screen, one gives a variable (a struct) with similar but not identical information. A better description of the two views and a reason why they come out that way would be useful

Reply:
To be fair, I would throw away "scientifc" output and would leave only the second one. But I am certainly looking into it as a programmers.

But historically, this is what was displayed by Horace/disp and this constrans the information useful for a scientist. (not sufficient for any other purposes) I can not explain why this selection was done and do not like it but as it was probably reason behind it, I am maintaining it.

Reply:
ok if there are good historical reasons for this, let it stay